### ISSUE TITLE
error while starting the server

### ISSUE THREAD

**Issue by Aqonn** [Wizelf402/Permafrost#73]
![image](https://user-images.githubusercontent.com/118808713/203279781-3d5a77ef-1dd3-4324-9769-39e9d500d6be.png)
how can i solve this issue?

**Comment by NevadaActual** [comment]
Hi! I haven't seen this issue before, which is interesting. It may have something to do with the fact the start.bat is being started with powershell instead of command prompt? But the first and foremost suggestion I have is to try and fresh install the modpack from https://www.curseforge.com/minecraft/modpacks/permafrost-eots/files/4017145. If that has no affect, try to install java 11, as java 18 has many different deprecated features removed entirely that may affect the mods running.

**Comment by Aqonn** [comment]
Hi! Well the start file is start.ps1 for me so thats why i started it with powershell, i tried reinstalling and with java 11 but its still the same error

**Comment by NevadaActual** [comment]
So, just to confirm, the file is start.ps1 when you download it for the first time? Can you try renaming the file to start.bat and see if it works then? Also, can you upload the contents of your start.ps1 to here so I can see if it matches what the official download has?

**Comment by Aqonn** [comment]
> So, just to confirm, the file is start.ps1 when you download it for the first time? Can you try renaming the file to start.bat and see if it works then? Also, can you upload the contents of your start.ps1 to here so I can see if it matches what the official download has?

yeah it was .ps1 from the start, i tried renaming but nothing changed
sorry but this was my best idea to how to upload the thing
[start.ps1 content.txt](https://github.com/Wizelf402/Permafrost/files/10090266/start.ps1.content.txt)

**Comment by NevadaActual** [comment]
Okay, it looks like the serverpack creator switched their default start.bat file quite a bit from the copy I have on my machine. Can you try to see if this one works? The issue (according to what I found online) has to do with no argument being passed into the Java command, and the only way I can think of that happening is something weird with the setup of the .ps1 file.

[start.bat.txt](https://github.com/Wizelf402/Permafrost/files/10099614/start.bat.txt)
